elements/pl-integer-input/pl-integer-input.py

```diff
@@ -215,6 +228,8 @@ def parse(element_html, data):
         a_sub_parsed = pl.string_to_integer(str(a_sub), base)
         if a_sub_parsed is None:
             raise ValueError('invalid submitted answer (wrong type)')
+        if a_sub_parsed > 2 ** 53 - 1 or a_sub_parsed < -((2 ** 53) - 1):
+            data['format_errors'][name] = 'Answer must be between (2**53)-1 and -((2**53)-1)'
```

Comment:
This error will be shown to students, some who may not be familiar with Python's representation of exponentiation. I'd suggest showing the actual numbers here as well, in a similar fashion as Matt's suggestion above.